packages/client/utils/AtlassianManager.ts

```diff
@@ -831,4 +923,87 @@ export default abstract class AtlassianManager {
       `https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/screens/${screenId}/tabs/${tabId}/fields/${fieldId}`
     )
   }
+
+  async registerWebhooks(cloudId: string, webhooks: WebhookDetails[]) {
+    return this.post(`https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/webhook`, {
+      url: makeAppURL('http://localhost:3000', '/webhooks/jira'),
+      // url: makeAppURL(appOrigin, '/webhooks/jira'),
```

Comment:
+1 that line should be correct, maybe you're missing some `.env` variables?
```
HOST='localhost'
PORT='3000'
PROTO='http'
```

Reply:
probably i set it aside to investigate other details